client/packages/common/src/ui/forms/JsonForms/useDocumentLoader.ts

```diff
@@ -84,7 +81,7 @@ export const useDocumentLoader = (
     isLoading,
     error,
     documentId,
-    data,
+    data: createDoc?.data ?? databaseResponse?.data,
```

Comment:
I needed to use the createDoc.data if the patient is created

Reply:
ah, sorry. missed that one..